spec/models/service_provider_spec.rb

```diff
@@ -70,13 +67,35 @@
     end
   end
 
+  describe 'piv_cac_available?' do
+    context 'when the service provider is with an enabled agency' do
+      it 'is truthy' do
+        allow(Figaro.env).to receive(:piv_cac_agencies).and_return(
+          [service_provider.metadata[:agency]].to_json
```

Comment:
Should we replace `metadata[:agency]` with `agency`?

Reply:
Probably. I'll fix that when I revert the ordering from the previous commit. It broke more tests and feels like feature creep since that's not part of account creation.